index.html

```diff
@@ -7,13 +7,103 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
-<body>
+<body class="container">
+            <header class="row">
+                <h3>Project 1: A small bussiness </h3>
+                <nav class="col-12 row">
```

Comment:
Try to avoid mixing `row` and `col-*` classes. An element should either be a row, or a column, e.g.:

```html
<nav class="row">
  <div class="col-8">
    <div class="row">
      ...
    </div>
  </div>
  <div class="col-4">
    ...
  </div>
</nav>
```